{{cookiecutter.project_name}}/docs/template_usage.md

```diff
@@ -150,10 +154,20 @@ The following pre-commit hooks are for code style and format:
     black on Python code in docs.
 -   [prettier](https://prettier.io/docs/en/index.html):
     standard code formatter for non-Python files (e.g. YAML).
+-   [ruff][] based checks:
+    -   [isort](https://beta.ruff.rs/docs/rules/#isort-i) (error code: `I`):
```

Comment:
For consistency, we should use Ruff terminology:

![grafik](https://user-images.githubusercontent.com/291575/220868129-ef8470ae-6da5-46ef-81fe-0b7dde34ef5d.png)

- `E`, `F`, …: “rule category” or “rule prefixes”

  ![grafik](https://user-images.githubusercontent.com/291575/220867176-2bac7394-b53b-4409-bbf8-1fa6cfc491d9.png)

- `E402`, …: “rule code”

  ![grafik](https://user-images.githubusercontent.com/291575/220868259-a2a3dbb9-21c4-4703-ae49-91abc6e4f9cc.png)

Reply:
I fixed it. Leaving this unresolved for acknowledgement